config/initializers/csp.rb

```diff
@@ -1,8 +1,11 @@
+gon_gem_sha = "'sha256-+PC+Kf6qxwFk9MeuRMDmsALBNWNX473zplWkiVYhJgY='"
+popover_sha = "'sha256-1kYydMhZjhS1eCkHYjBthAOfULylJjbss3YE6S2CGLc='"
+
 SecureHeaders::Configuration.default do |config|
   config.csp = {
     preserve_schemes: true, # default: false.
     default_src: %w('self'),
-    script_src: %w('self' 'unsafe-eval'),
+    script_src: ["'self'", "'unsafe-eval'", gon_gem_sha, popover_sha],
```

Comment:
I poked at it a few diff ways and thought this was marginally more readable. None of em are super fantastic though.